README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/yeoman/yeoman-app.svg)](https://travis-ci.org/yeoman/yeoman-app)
```

Comment:
Let's do the badge on the same line as the title? ([example](https://github.com/yeoman/generator-angular/blob/master/readme.md))